docs/process/Testing.md

```diff
@@ -0,0 +1,43 @@
+**Scenario:** You are testing a PR for an issue
+
+- Checkout the associated branch, and make sure you are on its latest commit
+- If the issue has another issue associated with it
+  - make sure you are testing the other issue at the same time 
+    - example: a new screen for managing orders requires an additional endpoint on the backend
+  - otherwise, make sure your backend/frontend is on the latest master version
+    - the database state should likewise match the latest master version as closely as possible (before migrations and such)
+
+- For backend issues:
+  - run any required database migrations first
+  - run `mvn clean package`
+  - If there are endpoint migrations:
+    - start the application and login
+    - obtain the auth token from the `Sessions` table in the database
+    - run the following command in a terminal:
+      - `curl -X GET -H "X-Auth-Token: <token from table>" localhost:8080/migration/<endpoint name>`
+
+- As an additional step when testing backend issues
```

Comment:
does this need to be a seperate section?

Reply:
I tried having it not be a separate section but it felt janky; this felt more natural to read (to me at least)

Reply:
what if you have a `Backend Steps` section, with several sub-parts? The migration steps are a bit weird too, since you say `run any required database migrations first` and then `If there are endpoint migrations`